deeppavlov/core/agent_v2/agent.py

```diff
@@ -31,18 +34,24 @@ def __call__(self, utterances: Sequence[str], user_telegram_ids: Sequence[Hashab
 
         state = self.state_manager.get_state(me_dialogs)
 
-        self._update_utterances(me_dialogs, state)
+        selected_skills = self.skill_manager.get_skill_responses(state)
+        self._update_utterances(me_dialogs, selected_skills, key='selected_skills')
 
-        skill_names, utterances, confidences, profiles = self.skill_manager(state)
+        skill_names, responses, confidences, profiles = self.skill_manager(state)
 
         self._update_profiles(me_users, profiles)
 
-        self.state_manager.add_bot_utterances(me_dialogs, utterances, [datetime.utcnow()] * len(me_dialogs),
+        self.state_manager.add_bot_utterances(me_dialogs, responses, responses,
+                                              [datetime.utcnow()] * len(me_dialogs),
                                               skill_names, confidences)
 
         self._update_annotations(me_dialogs)
```

Comment:
It looks like now it makes more sense to annotate 'sent_text', not 'text' in bot utterances.